media/public/include/MediaCommon.h

```diff
@@ -121,6 +131,16 @@ enum class PlaybackState
     FAILURE        /**< The backend player failed to set playback state. */
 };
 
+/**
+ * @brief Audio specific configuration
+ */
+struct AudioConfig
+{
+    uint32_t numberOfChannels = kInvalidAudioChannels; /**< The number of channels. */
+    uint32_t sampleRate = kInvalidAudioSampleRate;     /**< The sampling rate.*/
+    std::vector<uint8_t> codecSpecificConfig;          /**The audio specific config. Zero length if no specific config*/
```

Comment:
minor, and I'm not sure myself and I know this is what I specified, but from an API point of view do you think it would make more sense to call this mimeSpecificConfig?

Reply:
yes make sense, but I'm not quite sure it if it is better